bundles/net.tourbook/src/net/tourbook/ui/tourChart/SlideoutTourChartOptions.java

```diff
@@ -101,6 +102,8 @@
    private Button    _chkShowValuePointTooltip;
    private Button    _chkSelectAllTimeSlices;
 
+   private Scale     _scaleNightSectionsOpacity;
```

Comment:
I try to use a scale control only when necessary, you will find them mostly in old MT code. 

Advantages of the spinner controls:

* The value is just displayed and must not be displayed additional (when it's useful)
* With `<Ctrl>` `<Shift>` you can accelerate it with the mouse

![mt spinner vs scale](https://user-images.githubusercontent.com/1283445/113722636-e6987280-96f0-11eb-85d5-2eb28b8d1950.png)

Reply:
Ok, I will change it so a [Spinner](https://help.eclipse.org/2021-03/index.jsp?topic=/org.eclipse.platform.doc.isv/reference/api/org/eclipse/swt/widgets/Spinner.html) control

Reply:
Fixed in next PR